Gopkg.toml

```diff
@@ -34,15 +34,17 @@ required = ["github.com/wadey/gocovmerge"]
 [[override]]
   name = "github.com/containerd/containerd"
   source = "github.com/simonferquel/containerd"
-  revision = "42c3614d58767e96c1752977fed94d0b140a0173"
+  branch = "push-can-mount"
```

Comment:
Simon has pushed the commit back, you should switch back to the commit (else changes to the branch will break things).

Reply:
Fixed